### ISSUE TITLE
e2e cypress quickstarter test results mismatch (4expected 2 found)

### ISSUE THREAD

**Issue by clemensutschig** [opendevstack/ods-quickstarters#603]
**To Reproduce**
latest logs (master) on ods-core/readme

**Affected version (please complete the following information):**
 - OpenShift: [3.11]
 - OpenDevStack [master]

**Log Output (ensure to remove any confidential information like tokens, project names, etc.**
```
[0;32m    amazon-ebs: Verifying unit tests of run-unitt-cypress-unitt-master-1 ...[0m
[0;32m    amazon-ebs:     quickstarter_test.go:346: Expected 4 unit tests, but found only 2 for build:run-unitt-cypress-unitt-master-1[0m
```

looks like at least one spec is not executed?!
```
[0;32m    amazon-ebs: ────────────────────────────────────────────────────────────────────────────────────────────────────[0m
[0;32m    amazon-ebs:[0m
[0;32m    amazon-ebs:   Running:  integration.spec.js                                                             (1 of 1)[0m
[0;32m    amazon-ebs:[0m
[0;32m    amazon-ebs:   (Results)[0m
[0;32m    amazon-ebs:[0m
[0;32m    amazon-ebs:   ┌────────────────────────────────────────────────────────────────────────────────────────────────┐[0m
[0;32m    amazon-ebs:   │ Tests:        0                                                                                │[0m
[0;32m    amazon-ebs:   │ Passing:      0                                                                                │[0m
[0;32m    amazon-ebs:   │ Failing:      0                                                                                │[0m
[0;32m    amazon-ebs:   │ Pending:      0                                                                                │[0m
[0;32m    amazon-ebs:   │ Skipped:      0                                                                                │[0m
[0;32m    amazon-ebs:   │ Screenshots:  0                                                                                │[0m
[0;32m    amazon-ebs:   │ Video:        false                                                                            │[0m
[0;32m    amazon-ebs:   │ Duration:     0 seconds                                                                        │[0m
[0;32m    amazon-ebs:   │ Spec Ran:     integration.spec.js                                                              │[0m
[0;32m    amazon-ebs:   └────────────────────────────────────────────────────────────────────────────────────────────────┘[0m
``` 

maybe a temp hickup - but here to track..

Expected tests:
https://github.com/opendevstack/ods-quickstarters/blob/master/e2e-cypress/testdata/steps.yml#L15

**Comment by clemensutschig** [comment]
@cschweikert @m-apsolon can you take a look?

**Comment by clemensutschig** [comment]
I am wondering, if there is now logic to skip a truly empty test ..

https://github.com/opendevstack/ods-quickstarters/blob/master/e2e-cypress/files/tests/integration/integration.spec.ts

vs.

https://github.com/opendevstack/ods-quickstarters/blob/master/e2e-cypress/files/tests/installation/installation.spec.ts

**Comment by clemensutschig** [comment]
and I am still trying to find out why this is 4 instead of 2 .. ?!

**Comment by clemensutschig** [comment]
@herrkoch

**Comment by cschweikert** [comment]
Hi, the issue came with https://github.com/opendevstack/ods-quickstarters/pull/601

**Comment by cschweikert** [comment]
3 tests were replaced with 1 -> -2 tests

**Comment by clemensutschig** [comment]
@cschweikert thanks much .. @herrkoch did you NOT run the tests beforehand?!

**Comment by herrkoch** [comment]
Oops, sorry, I thought these tests run automatically as part of the checks when I open a PR. :-x Thanks for fixing @clemensutschig !

**Comment by clemensutschig** [comment]
@herrkoch the CI to build the AMI runs the tests full scale - you need to run make test with the quickstarter ... because a full ods installation is reqired ... I fixed the regression in #604 🗡️

**Comment by herrkoch** [comment]
Okay thanks, will keep this in mind. I guess I have to wear that hat now for breaking master... 🎩 😉
